src/Octoshift/GithubApi.cs

```diff
@@ -515,20 +490,24 @@ public virtual async Task<Mannequin> GetMannequin(string orgId, string username)
                     data => (JObject)data["data"]["node"]["mannequins"]["pageInfo"])
                 .FirstOrDefaultAsync(jToken => username.Equals((string)jToken["login"], StringComparison.OrdinalIgnoreCase));
 
-            return mannequin is null
-                ? new Mannequin()
-                : new Mannequin
-                {
-                    Id = (string)mannequin["id"],
-                    Login = (string)mannequin["login"],
-                    MappedUser = mannequin["claimant"].Any()
-                    ? new Claimant
-                    {
-                        Id = (string)mannequin["claimant"]["id"],
-                        Login = (string)mannequin["claimant"]["login"]
-                    }
-                    : null
-                };
+            return mannequin is null ? new Mannequin() : MannequinBuilder(mannequin);
+        }
+
+        public virtual async Task<IEnumerable<Mannequin>> GetMannequins(string orgId)
+        {
+            var url = $"{_apiUrl}/graphql";
+
+            var payload = GetMannequinsPayload(orgId);
+
+            return await _client.PostGraphQLWithPaginationAsync(
+                    url,
+                    payload,
+                    data => (JArray)data["data"]["node"]["mannequins"]["nodes"],
+                    data => (JObject)data["data"]["node"]["mannequins"]["pageInfo"])
+                .Select(mannequin => MannequinBuilder(mannequin)
+
+                )
+                .ToListAsync();
```

Comment:
Based on my suggestion for `GetMannequin` the `ToListAsync` should be removed so that we will return the actual `IEnumerable` object.